darts/models/forecasting/forecasting_model.py

```diff
@@ -2106,6 +2115,57 @@ def _optimized_historical_forecasts(
         )
         return []
 
+    @staticmethod
+    def _inverse_transform_forecasts_list(
+        forecasts_list_transformed: Union[List[TimeSeries], List[List[TimeSeries]]],
+        scaler: Scaler,
+    ) -> Union[List[TimeSeries], List[List[TimeSeries]]]:
+        """Applies inverse transform on the list (of lists) of forecasts.
+        Uses the scaler trained previously on the same set of series.
+
+        Parameters
+        ----------
+        forecasts_list_transformed
+            The list (of lists) of forecasts.
+        scaler
+            The scaler trained on the same set of series that was used to generate forecasts.
+
+        Returns
+        -------
+        Union[List[TimeSeries], List[List[TimeSeries]]]
+            A list (of lists) of inversely transformed forecasts.
+
+        """
+
+        if not forecasts_list_transformed:
```

Comment:
i don't get this point. If `forecasts_list_transformed` is empty you are doing nothing right?

Can you check that before calling the method? 

The method is private, so it should be used within this class.

Make it sense? :-)